addons/addon-base-raas/packages/base-raas-services/lib/storage-gateway/storage-gateway.js

```diff
@@ -0,0 +1,204 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+const Service = require('@aws-ee/base-services-container/lib/service');
+const { runAndCatch } = require('@aws-ee/base-services/lib/helpers/utils');
+
+const uuid = require('uuid/v1');
+let fetch = require('node-fetch');
+const storageGatewayDbRecord = require('../schema/storage-gateway-db-record');
+
+// Webpack messes with the fetch function import and it breaks in lambda.
+if (typeof fetch !== 'function' && fetch.default && typeof fetch.default === 'function') {
+  fetch = fetch.default;
+}
+
+const settingKeys = {
+  tableName: 'StorageGateway',
```

Comment:
The settings key in the settings file is 'dbStorageGateway', not sure if I missed a step here somewhere?

Reply:
'dbStorageGateway' variable in yml default file[1] is used for creating the table[2]. The settings key here is the table name suffix and we add a prefix here[3] where the prefix is the namespace [4]

[1] https://github.com/awslabs/service-workbench-on-aws/pull/180/files#diff-3482c65904df9038085c7c0dbcc67dc3d1c494e2b8f90562fe0d3fed951d8a59R178
[2] https://github.com/awslabs/service-workbench-on-aws/pull/180/files#diff-e6d549fa787ec38d72e853845a7bc58fc6decfa5ae79324a0178f7cd236f0501R1339-R1349
[3] https://github.com/awslabs/service-workbench-on-aws/blob/master/addons/addon-base-raas/packages/base-raas-rest-api/lib/plugins/services-plugin.js#L114-L117
[4] https://github.com/awslabs/service-workbench-on-aws/blob/master/main/config/settings/.defaults.yml#L95